.github/workflows/yki.yml

```diff
@@ -0,0 +1,70 @@
+name: YKI
+
+on:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - '**.md'
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Liittyy: https://github.com/marketplace/actions/checkout#fetch-all-history-for-all-tags-and-branches

Reply:
Tämä vastannee `.travis.yml`:ssä löytyvää asetusta `git.depth: false`. Ilman tätä ei vissiin ole tarpeeksi tietoa checkoutatusta versiosta. `lein-git-version`-plugini ei tällöin toimi kunnolla, oireina mm. varoitusviestejä konsoliin ja  testin `yki.handler.status-test/buildversion-test` failaus.

Reply:
Eri asia sitten on onko tuo buildversion-test tai APIsta tarjoiltu buildversion.txt lainkaan relevantti.